app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -47,4 +47,36 @@ def supported_catalog_types
   def self.display_name(number = 1)
     n_('Container Provider (OpenShift)', 'Container Providers (OpenShift)', number)
   end
+
+  #
+  # This provider supports alert labels, populated from the labels provided by Prometheus.
+  #
+  supports :alert_labels
+
+  #
+  # Returns the alert labels associated to the given alert status.
+  #
+  # @param alert_status [MiqAlertStatus] The alert status object.
+  # @return [Array<MiqAlertStatusLabel>] The array of labels.
+  #
+  def alert_labels(alert_status)
+    # Find the event that correspond to the given alert status:
+    events = EventStream.where(:ems_ref => alert_status.event_ems_ref)
```

Comment:
Since the EmsRef is made up from a checksum of alert labels + annotation + start time maybe we should add the ems_id to be on the safe side? It is not probably for two different alerts from two different ems's to have the same start ems_ref - labels & annotation can easily be empty but start time is usually unique but I would not want to rely on that

Reply:
Added the `ems_id` in the pull request for the Kubernetes provider.